CMakeLists.txt

```diff
@@ -71,7 +68,17 @@ if(GST_BUILD)
 endif(GST_BUILD)
 message(INFO "${GST_LIBRARIES}")
 
-include_directories(${CMAKE_CURRENT_BINARY_DIR}
+link_directories(${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+
+file(GLOB_RECURSE source_files ${sources_directory}/*.cpp ${include_directory}/*.hpp ${resources_directory}/*.qrc)
+
+add_executable(ia ${source_files})
```

Comment:
this should change to `dash`? (also the references below it)

when I built it, it created a binary called ia

Reply:
Yes, but not in this set of commits.  There's more work to be done here like killing GLOB, scrubbing include directory and link lib stuff.  I think we're including a lot of stuff that's not truly necessary.